kas/repos.py

```diff
@@ -316,10 +316,21 @@ async def apply_patches_async(self):
             elif os.path.isdir(path) \
                     and os.path.isfile(os.path.join(path, 'series')):
                 with open(os.path.join(path, 'series')) as f:
+                    ln = 0
                     for line in f:
-                        if line.startswith('#'):
+                        ln += 1
+                        if not line.rstrip('\n') or line.startswith('#'):
                             continue
-                        p = os.path.join(path, line.split(' #')[0].rstrip())
+                        pn = line.split(' #')[0].rstrip()
+                        if not pn:
+                            logging.error('Could not parse patch file name from a \'series\' file. '
+                                          '(file: %s, line: %d, repo: %s, patch entry: %s)',
+                                          os.path.join(path, 'series'),
+                                          ln,
+                                          self.name,
+                                          patch['id'])
+                            return 1
+                        p = os.path.join(path, pn)
                         if os.path.isfile(p):
```

Comment:
this line behaves strange on my linux with python ver. 3.10.7,

if `pn` contains space at the beginning (e.g. `  001.patch`), `isfile()` says that such path `p` exists, and error appears later when patch is about to be applied,